dialogs/EditVisgroups.cpp

```diff
@@ -0,0 +1,152 @@
+#include "dialogs/EditVisgroups.h"
+
+#include <QPushButton>
+#include <QLabel>
+#include <QHboxLayout>
+#include <QVboxLayout>
+#include <QGridLayout>
+#include <QGroupBox>
+#include <QStyleFactory>
+#include <QHeaderView>
+
+using namespace ui;
+
+CEditVisgroups::CEditVisgroups( QWidget *pParent ) :
+	QDialog( pParent )
+{
+	this->setWindowTitle( tr( "Edit Visgroups" ) );
+
+	auto pDialogLayout = new QGridLayout( this );
+
+	auto pGroupsLabel = new QLabel( tr( "Groups:" ), this );
+	m_pVisgroupTree = new QTreeWidget( this );
+	m_pVisgroupTree->setStyle( QStyleFactory::create( "Windows" ) );
```

Comment:
I'd rather leave this to a stylesheet. I think I can set a global style for all Qt windows in hammer.